### PR TITLE
Add New Account Card

### LINKED ISSUE #88: Implement the "Add new account" card
Image

Let's add it to the Widgets project. 

The card should be a link to another page. 
Also, it should have an optional `onClick` handler.

### PR DESCRIPTION
**Pull request by vitalii-ilchenko-ssa-group**
Closes #88

### CODE REVIEW

File: packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  onclick,
+}: AddNewAccountCardProps) => {
+  const theme = useTheme();
+
+  return (
+    <React.Fragment>
+      {typeof onclick === 'function' ? (
+        <Button onClick={onclick} css={[ResetBtnStyles, AccountCard]}>
+          <Icon name="plus" size={12} color={theme.colors.blueRoyal} />
+          Add new account
```
Comment (arthur-kashaiev-ssa-group) [inline packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx:19]: Let's not hardcode the text (here and below). Pass it via props.

**Review by arthur-kashaiev-ssa-group — CHANGES_REQUESTED** [review]

File: packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx (hunk 1 above)
Comment (vitalii-ilchenko-ssa-group) [reply packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx:19]: Done

File: packages/widgets/src/components/AddNewAccountCard/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { AddNewAccountCard } from './AddNewAccountCard';
```
[the remark this answers is not in the document]
Comment (vitalii-ilchenko-ssa-group) [reply packages/widgets/src/components/AddNewAccountCard/index.ts:1]: Done

File: packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        </Button>
+      ) : (
+        <Link to={`${link}`} css={AccountCard}>
```
[the remark this answers is not in the document]
Comment (vitalii-ilchenko-ssa-group) [reply packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx:22]: Done

File: packages/widgets/src/components/AddNewAccountCard/types.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export interface AddNewAccountCardProps {
+  link?: string;
```
[the remark this answers is not in the document]
Comment (vitalii-ilchenko-ssa-group) [reply packages/widgets/src/components/AddNewAccountCard/types.ts:2]: Done

File: packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useTheme } from '@emotion/react';
+import { Button, Icon } from '@ssa-ui-kit/core';
+import { AddNewAccountCardProps } from './types';
+import { AccountCard, ResetBtnStyles } from './styles';
+
+export const AddNewAccountCard = ({
+  link,
+  onclick,
```
[the remark this answers is not in the document]
Comment (vitalii-ilchenko-ssa-group) [reply packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx:10]: Done

File: packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+}: AddNewAccountCardProps) => {
+  const theme = useTheme();
+
+  return (
+    <React.Fragment>
```
[the remark this answers is not in the document]
Comment (vitalii-ilchenko-ssa-group) [reply packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx]: Done

[the remark this answers is not in the document]
Comment (arthur-kashaiev-ssa-group) [reply packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx]: @vitalii-ilchenko-ssa-group Please learn the example and apply the same approach to your task. 
You should wrap <Card /> with <Link /> and not the opposite.

**Review by react-developer-ssa-group — APPROVED** [review]

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 2  | ❗       |
    


*❗ storybook/components-card--icon-header__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/utility-responsive-image--responsive-image__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 8       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 2  | ❗       |
    


<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



*❗ custom/widgets-meal-nutrients--hint-opened__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-meal-nutrients--hint-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 1  | ❗       |
    


*❗ storybook/utility-responsive-image--responsive-image__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 8       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 4  | ❗       |
    


<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



*❗ custom/widgets-meal-nutrients--hint-opened__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-meal-nutrients--hint-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-progressInfo__[w900px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-calories--default__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |

File: packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (vitalii-ilchenko-ssa-group) [reply packages/widgets/src/components/AddNewAccountCard/AddNewAccountCard.tsx]: Fixed

**Review by arthur-kashaiev-ssa-group — CHANGES_REQUESTED** [review]

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 8       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 5  | ❗       |
    


<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



*❗ custom/widgets-meal-nutrients--hint-opened__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-meal-nutrients--hint-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-progressInfo__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-calories--default__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-meal-nutrients--default__[w900px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 1  | ❗       |
    


*❗ storybook/utility-responsive-image--responsive-image__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 8       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>❗ custom/widgets-linkstabbar--current-focus__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-meal-nutrients--hint-opened__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-meal-nutrients--hint-opened__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-navbar-opened__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-progressInfo__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-progressInfo__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Review by arthur-kashaiev-ssa-group — APPROVED** [review]

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 1  | ❗       |
    


*❗ storybook/utility-responsive-image--responsive-image__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 8       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 4  | ❗       |
    


<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--default__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-add-new-account-card--with-link__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



*❗ custom/widgets-meal-nutrients--hint-opened__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-meal-nutrients--hint-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-progressInfo__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-heart-rate--default__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Pull request merged**